src/features/SignAndVerifyMessage/SignAndVerifyMessage.tsx

```diff
@@ -70,10 +70,11 @@ const tabClickRedirect = (history: History<unknown>, url: string): void => {
   history.push(url);
 };
 
-function SignAndVerifyMessage(props: RouteComponentProps) {
+function SignAndVerifyMessage() {
   const [showSubtitle, setShowSubtitle] = useState(true);
 
-  const { history, location } = props;
+  const history = useHistory();
+  const location = useLocation();
```

Comment:
Prefer passing values as props to remove the need to mock the hooks in the tests and decorate in the stories.